crowdin_api/api_resources/bundles/resource.py

```diff
@@ -14,9 +14,11 @@ class BundlesResource(BaseResource):
     Link to documentation for enterprise:
     https://developer.crowdin.com/enterprise/api/v2/#tag/Bundles
     """
-
-    def get_bundles_path(self, projectId: int, bundleId: Optional[int] = None):
+    def get_bundles_path(self, projectId: int, bundleId: Optional[int] = None, exportId: Optional[str] = None):
```

Comment:
Could you create another method path? It is a good idea to decompose paths in different methods.